simulation/sub8_gazebo/nodes/pedometry.txt

```diff
@@ -1 +1 @@
-34989.8186173
\ No newline at end of file
+35424.0068616
```

Comment:
Remove this file from the git. Tracking a file that will be different for everyone, that's used to track how much its moved in sim is useless. Also, it makes it a pain to commit while the sim is running.